### ISSUE TITLE
[BUG] <title>urllib.error.HTTPError: HTTP Error 404: Not Found</title>

### ISSUE THREAD

**Issue by jbenr** [nflverse/nfl_data_py#104]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you installed the latest development version of the package(s) in question?

- [X] I have installed the latest development version of the package.

### If this is a data issue, have you tried clearing your nflverse cache?

I am not using an nflverse R package (nflreadr, nflfastR) to access this data.

### What version of the package do you have?

nfl_data_py==0.3.2

### Describe the bug

I'm getting a "urllib.error.HTTPError: HTTP Error 404: Not Found" when trying to run import_pbp_data for 2024. It works great for 2023... and all other years - so I don't think it's my code. I think there's a bug with 2024 data. 

### Reprex

```r
szn = 2024
dat = nfl.import_pbp_data([szn])
```

### Expected Behavior

I expect dat to return a df of play by play data. 

### nflverse_sitrep

```r
NA
```

**Comment by aggieed97** [comment]
I believe this is the same issue as described here: 

https://github.com/nflverse/nfl_data_py/issues/99#issuecomment-2342212809

No participation data anymore means you'll have to put the following `include_participation=False` in the function. Alec mentioned fixing this for the next update in the library.

**Comment by alecglen** [comment]
Fixed in 0.3.3. This will be available via PyPI soon, but can be installed now with the command below if needed.
```
pip install --force-reinstall nfl-data-py@git+https://github.com/nflverse/nfl_data_py.git@v0.3.3
```
